Mapping/zds.extraElementToZaakEigenschap.mapping.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "ZDSToZGWZaak",
-  "$id": "https://opencatalogi.nl/schemas/zds.extraElementToZaakEigenschap.schema.json",
+  "$id": "https://zds.nl/mapping/zds.extraElementToZaakEigenschap.mapping.json",
```

Comment:
Then all references have to be changed too

Reply:
done